src/mail/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "dependencies": {
+    "html-to-text": "8.2.0",
+    "mjml": "4.13.0"
+  },
+  "scripts": {
+    "build-mjml-to-html": "./bin/mjml-to-html",
+    "build-mails-html-to-plain-text": "./bin/html-to-plain-text",
+    "build-mails": "yarn build-mjml-to-html; yarn build-mails-html-to-plain-text;"
+  },
+  "repository": "git@github.com:openfun/joanie.git",
+  "author": "France Université Numérique",
+  "license": "MIT"
```

Comment:
For people who are using [volta](https://volta.sh/) (like me ^^), I also add a `volta` property. In this way, even if I run commands on my local machine, I use the right version of node.
```
"volta": {
    "node": "16.15.1"
  },
```

Reply:
perfect